.env

```diff
@@ -1,3 +1,8 @@
-EXTEND_ESLINT=true
+# Base URL path for the enviroment
+PUBLIC_URL = '/'
```

Comment:
This is the path recommended for local dev

Reply:
We could set it to something different if we want to avoid the common pitfall of assuming everthing will be hosted at '/' during dev. 

e.g. if PUBLIC_URL is '/foo/bar' the local dev server root would be `localhost:3000/foo/bar/`

Reply:
I agree, I've taken the liberty of making the default `/dev/` and making the `eslint`/`prettier` changes we previously discussed. Once the checks run again I will merge this PR.